drools-parser/src/main/java/org/drools/parser/DRLVisitorImpl.java

```diff
@@ -140,13 +159,18 @@ public AccumulateImportDescr visitImportAccumulateDef(DRLParser.ImportAccumulate
 
     @Override
     public FunctionDescr visitFunctiondef(DRLParser.FunctiondefContext ctx) {
+        // function String functionA(String s, Integer i) {
+        //    foo();
+        // }
```

Comment:
Example syntax as a comment. Is this too much? But maybe helpful for new contributors.